azure-out-of-cluster.md

```diff
@@ -29,23 +29,27 @@ The values needed to provide access to the Azure Storage Account where cost data
 
 With these values in hand, you can now provide them to Kubecost to allow access to the Azure Storage API.
 
-To create this secret you will need to create a JSON file that must be named azure-storage-config.json
+To create this secret you will need to create a JSON file that **must**  be named cloud-integration.json
 with the following format:
 
 ```
 {
-  "azureSubscriptionID": "<SUBSCRIPTION_ID>",
-  "azureStorageAccount": "<STORAGE_ACCOUNT_NAME>",
-  "azureStorageAccessKey": "<STORE_ACCESS_KEY>",
-  "azureStorageContainer": "<REPORT_CONTAINER_NAME>",
-  "azureContainerPath": "<AZURE_CONTAINER_PATH>",
-  "azureCloud": "<AZURE_CLOUD>"
+    "azure": [
+        {
+            "azureSubscriptionID": "AZ_cloud_integration_subscriptionId",
+            "azureStorageAccount": "AZ_cloud_integration_azureStorageAccount",
+            "azureStorageAccessKey": "AZ_cloud_integration_azureStorageAccessKey",
+            "azureStorageContainer": "AZ_cloud_integration_azureStorageContainer",
+            "azureContainerPath": "",
+            "azureCloud": "public/gov"
+        }
+    ]
 }
 ```
 
 Once you have the values filled out use this command to create the secret:
 
-`kubectl create secret generic <SECRET_NAME> --from-file=azure-storage-config.json -n kubecost`
+`kubectl create secret generic <SECRET_NAME> --from-file=cloud-integration.json -n kubecost`
 
 Once the secret is created, set `.Values.kubecostProductConfigs.azureStorageSecretName` to
```

Comment:
I think we can use 
Values.kubecostProductConfigs.cloudIntegrationSecret: "cloud-integration"
for all providers.

Reply:
Changed to cloudIntegrationSecre